config.w32

```diff
@@ -28,10 +28,8 @@ if(PHP_SOLR != 'no')
 		WARNING('solr was not enabled; IDN libraries (normaliz.lib) not found');
 		PHP_SOLR = "no";
 	}
-	if(!((CHECK_LIB('libssl.lib', 'solr', PHP_SOLR)
-		&& CHECK_LIB('libcrypto.lib', 'solr', PHP_SOLR)) ||
-		(CHECK_LIB('ssleay32.lib', 'solr', PHP_SOLR)
-		&& CHECK_LIB('libeay32.lib', 'solr', PHP_SOLR)))) {
+	if(!(CHECK_LIB('libssl.lib;ssleay32.lib', 'solr', PHP_SOLR)
+		&& CHECK_LIB('libcrypto.lib;libeay32.lib', 'solr', PHP_SOLR))) {
```

Comment:
As of PHP 7.1.0, we could use:
````.js
if (SETUP_OPENSSL('solr', PHP_SOLR)) {
````
It might make sense to add this right away, even if older PHP versions are still supported.

Reply:
I will look into that

Reply:
I still build PHP 7.0 (from Remi's security backport repo) every once in a while. Then the check does not become easy.
Something like this example for the SSH2 extension.
https://github.com/php/pecl-networking-ssh2/blob/master/config.w32#L18-L29
```
    if (typeof SETUP_OPENSSL === "function") {
	    var ret = SETUP_OPENSSL("ssh2", PHP_PHP_BUILD + "\\lib");
	    if (ret < 1) {
                WARNING("ssh2 support can't be enabled, openssl library not found")
                ssh2_all_ok = false;
	    }
    } else {
        if (!CHECK_LIB("libeay32.lib", "ssh2", PHP_PHP_BUILD + "\\lib")) {
            WARNING("ssh2 support can't be enabled, openssl library not found")
            ssh2_all_ok = false;
        }
    }

```

Reply:
Yeah, something like this (or drop support for PHP 7.0).